src/VueGAPI/GoogleAuthService.js

```diff
@@ -99,7 +99,7 @@ export default class GoogleAuthService {
 
   _setSession (response) {
     const profile = this.authInstance.currentUser.get().getBasicProfile()
-    const authResult = response.Zi
+    const authResult = this.authInstance.currentUser.get().getAuthResponse(true)
```

Comment:
Confirmed that this change fixes the problem for me @hanliao @cedpoilly

Reply:
This also fixes the issue for me